bot/cogs/reminders.py

```diff
@@ -148,6 +148,19 @@ async def _delete_reminder(self, reminder_id: str, cancel_task: bool = True) ->
             # Now we can remove it from the schedule list
             self.scheduler.cancel(reminder_id)
 
+    async def _edit_reminder(self, reminder_id: int, payload: dict) -> dict:
```

Comment:
This didn't really need to be a separate function since it's short and only used by `edit_reminder`, but it's fine honestly.